src/applications/mhv-secure-messaging/util/recipients.js

```diff
@@ -0,0 +1,18 @@
+export const getUniqueTriageGroups = threadList => {
+  const uniqueHash = threadList.reduce((acc, thread) => {
+    if (!acc[thread.recipientId]) {
+      acc[thread.recipientId] = {
+        triageGroupName: thread.triageGroupName,
+        sentDate: thread.sentDate,
+        triageGroupId: thread.recipientId,
+      };
+    }
+    return acc;
+  }, {});
+
+  const groups = Object.values(uniqueHash);
+  // sort the unique triage groups by sentDate in descending order
+  groups.sort((a, b) => new Date(b.sentDate) - new Date(a.sentDate));
```

Comment:
do we need to add `.sort` if we are eventually deriving only the count ?

Reply:
For this PR no, 

But for the vision of this feature we will. See context added on the PR.